cpp/oneapi/dal/algo/pca/backend/cpu/infer_kernel.cpp

```diff
@@ -51,16 +51,19 @@ static result_t call_daal_kernel(const context_cpu& ctx,
 
     const auto daal_data = interop::convert_to_daal_table<Float>(data);
     const auto daal_eigenvectors = interop::convert_to_daal_table<Float>(model.get_eigenvectors());
+    const auto daal_means = interop::convert_to_daal_table<Float>(model.get_means());
+    const auto daal_eigenvalues = interop::convert_to_daal_table<Float>(model.get_eigenvalues());
+
     const auto daal_result =
         interop::convert_to_daal_homogen_table(arr_result, row_count, component_count);
 
     interop::status_to_exception(
         interop::call_daal_kernel<Float, daal_pca_transform_kernel_t>(ctx,
                                                                       *daal_data,
                                                                       *daal_eigenvectors,
+                                                                      daal_means.get(),
                                                                       nullptr,
-                                                                      nullptr,
-                                                                      nullptr,
+                                                                      daal_eigenvalues.get(),
                                                                       *daal_result));
 
```

Comment:
is it possible to extend infer_results with adding singular values and explained variances, for getting all necessary results on cpp side?